### ISSUE TITLE
Question about pooling and normalization.

### ISSUE THREAD

**Issue by zhiqihuang** [UKPLab/sentence-transformers#1381]
Suppose I use a pre-trained sentence transformer model as a building block for a large model. For example, first I encode with `all-mpnet-base-v2` then feed the output to another neural network on top of it. During training, the gradient passes through all layers (including the sentence transformer)

In this case, should I disable the normalization layer? How normalization affects the gradient desent?

In detail, should I only perform pooling (the first `sentence_embeddings`)?
```
# Perform pooling
sentence_embeddings = mean_pooling(model_output, encoded_input['attention_mask'])

# Normalize embeddings
sentence_embeddings = F.normalize(sentence_embeddings, p=2, dim=1)
```

Thank you!

**Comment by nreimers** [comment]
You can keep the layer, no need to disable it.
